src/lib/strapi.ts

```diff
@@ -9,7 +9,7 @@ import axios, {
 import defu from "defu";
 import qs from "qs";
 import Cookies from "js-cookie";
-import ufo from "ufo";
+import { cleanDoubleSlashes, joinURL } from "ufo";
```

Comment:
Thanks for your PR ☺️

I already tested this but it will break on webpack application as the .mjs is not listed in module for some reason (nuxt2 application have this issue).

We have 2 choices:
- Document the issue in the doc so user can add the specific config in their webpack but can lead to issue/bug report for misreading the doc
- Make work the use of default so we don't have any issue thanks importing all as module name.

```suggestion
import * as ufo from "ufo";
```